lib/systems/inspect.nix

```diff
@@ -8,8 +8,12 @@ let abis = lib.mapAttrs (_: abi: builtins.removeAttrs abi [ "assertions" ]) abis
 
 rec {
   patterns = rec {
+    isi386         = { cpu = cpuTypes.i386; };
```

Comment:
Do you expect needing these? My thinking would be we would just use x86_32 as much as possible. We are doing a similar thing with isAarch32 as well. Even without these you can always do:

```
stdenv.hostPlatform.parsed.cpu.name == "i386"
```

if you really need a condition on a certain micro architecture.

Reply:
I don't.  I put them in to be symmetric with `isi686`.